src/cmds/scripts/pbs_init.d.in

```diff
@@ -499,7 +488,6 @@ stop_pbs() {
     fi
     if [ "${PBS_START_MOM}" -gt 0 ] ; then
       if check_prog "mom" ; then
-	site_mom_cleanup
```

Comment:
Please remove the definition of site_mom_cleanup. Nothing else references it. Also, please change the tabs to spaces in the next couple lines. It makes the indentation look incorrect on GitHub.

Reply:
I couldn't find any references to site_mom_cleanup in this file.  I did change the tabs to spaces as you requested.

Reply:
My bad. You already deleted the site_mom_cleanup definition.